libtiny_client/src/state.rs

```diff
@@ -232,6 +271,31 @@ impl StateInner {
                 }
             }
 
+            // Reply 477 when user needs to be identified with NickServ to join a channel
+            Reply { num: 477, params } => {
```

Comment:
Could you add a comment here showing the full message format. I had to check and it looks like

```
Msg { pfx: Some(Server("helix.oftc.net")), cmd: Reply { num: 477, params: ["osa1_", "#openjdk", "Cannot join channel (Need to be identified and verified to join this channel, \'/msg NickServ help\' to learn how to register and verify.)"] } }
```

So first arg is our nick, second is channel, then a message.